csrc/scheduler/normalization_utils.cpp

```diff
@@ -722,5 +722,26 @@ getOptionalInnerOuterPersistentBufferBatches(
   }
 }
 
+TensorView* getReferenceReductionTv(
```

Comment:
Could you add a comment describing what tensor should be a reference?

Reply:
added.